eng/pipelines/apiview-review-gen-swagger.yml

```diff
@@ -0,0 +1,56 @@
+pr: none
+
+trigger: none
+
+pool:
+  name: azsdk-pool-mms-win-2019-general

```

Comment:
Can this be ubuntu?

Reply:
Yes, It can be. I will file an issue to switch it to Ubuntu as separate change.